packages/table/src/column.tsx

```diff
@@ -5,11 +5,20 @@
  * and https://github.com/palantir/blueprint/blob/master/PATENTS
  */
 
-import { IProps } from "@blueprintjs/core";
 import * as React from "react";
+
+import { IProps } from "@blueprintjs/core";
+
 import { emptyCellRenderer, ICellRenderer } from "./cell/cell";
 import { IColumnHeaderRenderer, IColumnNameProps } from "./headers/columnHeaderCell";
 
+export type ColumnLoading = "header-loading"
+    | "cell-loading";
+export const ColumnLoading = {
```

Comment:
no, this is valid code; we see it a lot in autogenerated TS code from back end services. TypeScript allows you to overload names as long as they are used in different positions: _type_ positions and _value_ positions in the syntax tree.